app/models/project.js

```diff
@@ -50,6 +50,15 @@ export default Model.extend({
    */
   customer: belongsTo("customer"),
 
+  /**
+   * Whether the project's comments are visible to the customer
+   *
+   * @property customerVisible
+   * @type {Boolean}
+   * @public
+   */
+  customerVisible: attr("boolean", { defaultValue: false }),
```

Comment:
Refactor the whole model to class syntax if you touch it..

Reply:
What do you mean exactly by that?
Something like that:

```
export defautl class Employment extends Model {

...

customer = belongsTo("customer");

customerVisible = attr("boolean");
}

Reply:
Not exactly: https://guides.emberjs.com/release/models/defining-models/#toc_defining-attributes

Reply:
Same as https://github.com/adfinis-sygroup/timed-frontend/pull/562#discussion_r657992139